packages/mobile/package.json

```diff
@@ -174,6 +174,7 @@
     "postinstall-prepare": "^1.0.1",
     "react-devtools": "^3.6.0",
     "react-dom": "16.9.0",
+    "react-native-bundle-visualizer": "^2.1.1",
```

Comment:
Curious what this revealed. Did you mean to leave it in? If so, should we add a yarn script for it?